### DISCUSSION TITLE
Custom Styling

### DISCUSSION THREAD

**Discussion by bakarbsa** [rappasoft/laravel-livewire-tables#1041] [Q&A] [answered]
Hello, I've read the documentation but I'm still confused about custom styling, for example changing the style of the search column (border, background, icon, etc.)

**Comment by lrljoe** [comment, marked as answer]
To customise the Search Bar in particular, you'd need to publish the components.tools.toolbar blade 

Other elements such as tr, td, th etc are customisable via the Configure() function - [See Here](https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods)

I do agree that it should be possible to add custom classes into the Search box though, so I'll take a look when I get a moment!

**Reply by bakarbsa** [reply]
Thanks for the response! I've tried edit publish view and managed to customize the search bar if you find a way to customize it via Configure() that would be great! Thanks for the effort.

After trying the filter feature, I found the problem that the filter button couldn't be clicked, and I suspected that there was a problem in Alphine Js, and when I tried to make a normal button using Alphine Js, it didn't work. Do you have any suggestions for installing Alpine Js correctly on Laravel 9? I have followed the documentation though.

**Comment by lrljoe** [comment]
Are you getting any errors in your browser console?

I'm assuming you've done the relevant requires/includes and starts in your app.js, and run your webpack of choice and built your code? (E.g. npm run build or npm run dev if you're running a vote dev server).

If that all sounds a bit weird, then Livewire/Jetstream starter pack let's you kickstart your environment pretty cleanly with alpinejs etc all ready for you to go



Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
